sharktank/sharktank/types/pipelining.py

```diff
@@ -59,8 +59,8 @@ def iter_equal(A, B):
 
 
 def transfer_between_blocks(
-    x: AnyTensor, curr_block_tensors: dict[str, dict[str, AnyTensor]]
-) -> ShardedTensor:
+    *xs: AnyTensor | None, curr_block_tensors: dict[str, dict[str, AnyTensor]]
```

Comment:
Drop the `None` support. Its not really useful and makes the usecase more complicated.

Reply:
I have the None support since certain things in lllm.prefill are labelled as optional (e.g. `start_position` and `attention_mask`). I can handle those at the call site, but that will make it even longer there.